src/Microsoft.Crank.Models/Job.cs

```diff
@@ -233,6 +233,7 @@ public Source Source
         public RollingLog Output { get; set; } = new RollingLog(10000);
         public bool SelfContained { get; set; }
         public string BeforeScript { get; set; }
+        public string DockerInitScript { get; set; }
```

Comment:
Why not `InitScript` and also use it for non-docker path

Reply:
Wasn't sure what level we would want this at but InitScript makes sense. Made those changes.